semantic.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:       2.4
 
 name:                semantic
-version:             0.6.0.0
+version:             0.7.0.0
```

Comment:
This appears to be a backwards-compatible bugfix, so I think this could be 0.6.0.1?

Reply:
I meant to bump this on #212 since that added some new cmd line args, but didn't push the commit before that PR merged.